src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -30,11 +30,26 @@ public ArmSubsystem() {
         arm1Motor.setSmartCurrentLimit(60);
         arm2Motor.setSmartCurrentLimit(60);
 
+        arm1Motor.getEncoder().setPositionConversionFactor(360/32); // TODO: Check Conversion factor
+        arm2Motor.getEncoder().setPositionConversionFactor(360/32);
```

Comment:
unused for now, may want to comment until we start testing allocating them onto the sparks.